.github/workflows/lint_python.yml

```diff
@@ -6,21 +6,18 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
-      - run: |
-           sudo apt-get -q update
-           sudo apt-get -qq upgrade
-           sudo apt-get -qq install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils
+      # - run: sudo apt-get -q update  # Takes too long!
```

Comment:
I do not agree with this. Either we want packages to be up-to-date, then updating the lists is strictly required, or we do not want it, then the below `sudo apt-get -qq upgrade` is obsolete as well. I would however vote for always having updated packages, even that it takes a little longer.